lib/utils/KalturaUtils.js

```diff
@@ -301,6 +301,10 @@ KalturaUtils = {
 	},
 
 	downloadHttpUrl : function(urlStr, options, successCallback, errorCallback) {
+		if (!urlStr || (typeof urlStr !== 'string' && urlStr instanceof String)){
```

Comment:
don't understand this validation: typeof urlStr !== 'string' && urlStr instanceof String